cloudhub-2/modules/ROOT/pages/ch2-architecture.adoc

```diff
@@ -156,13 +156,15 @@ For example, if you deploy an application named `myapp` to Canada (Central), the
 
 CloudHub 2.0 backend services determine the values of:
 
-_uniq-id_::
+`uniq-id`::
 A 6-digit value appended to the app name to ensure uniqueness.
-_shard_::
-A 6-digit value associated with the space (private or shared) that the app is deployed to.
+`shard`::
+A 6-digit value associated with the private space that the app is deployed to.
```

Comment:
per Ana, we should provide examples here.

Reply:
@sarathecoubian can the docs team work on that to add the example as part of the new child work i have created as part of the investigation? https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001VkuYaYAJ/view